tests/meson.build

```diff
@@ -148,14 +148,14 @@ if gtest_dep.found()
     if nnstreamer_edge_support_is_available
       unittest_edge = executable('unittest_edge',
         join_paths('nnstreamer_edge', 'unittest_edge.cc'),
-        dependencies: [nnstreamer_unittest_deps],
+        dependencies: [gstedge_dep, nnstreamer_unittest_deps],
         install: get_option('install-test'),
         install_dir: unittest_install_dir
       )
 
       test('unittest_edge', unittest_edge, env: testenv)
     endif
-    
+
```

Comment:
Don't add unrelated contents

Reply:
Removed unrelated contents. Thank you :)